ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -55,6 +55,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.12"
+  source:
```

Comment:
`openshift-e2e-test-qe` does not reference `openshift-extended-upgrade-pre`, so we do not need the setting here.